### PR TITLE
Additional files now at 100% coverage

### PR DESCRIPTION
**Pull request by LawrenceMichaelMargaja**
string_utils.go is also now at 100% coverage.
files.go is at 85% coverage but I could not see anything else that could be tested in this file.

### CODE REVIEW

File: internal/utils_common/files_test.go [hunk 1]
```diff
@@ -5,11 +5,17 @@ import (
...
 	"os"
 	"path/filepath"
 	"testing"
 )
 
+/**
+go get github.com/golang/mock/gomock
+go install github.com/golang/mock/mockgen
+*/
+
```
Comment (dembygenesis) [inline internal/utils_common/files_test.go:18]: please remove

File: internal/utils_common/files_test.go [hunk 2]
```diff
@@ -220,6 +243,8 @@ func TestCopyDir_Success(t *testing.T) {
 		WipeDestinationExclusions: wipeDestinationExclusions,
 	}
 
+	fmt.Println("the opts --- ", opts.Source)
+
```
Comment (dembygenesis) [inline internal/utils_common/files_test.go:247]: please remove

File: internal/utils_common/files_test.go (hunk 1 above)
Comment (LawrenceMichaelMargaja) [reply internal/utils_common/files_test.go:18]: oh ok ok on it

File: internal/utils_common/files_test.go (hunk 2 above)
Comment (LawrenceMichaelMargaja) [reply internal/utils_common/files_test.go:247]: on it

**Comment by dembygenesis** [comment]
Thanks!

**Comment by dembygenesis** [comment]
nice work

**Pull request merged**